pkg/descheduler/strategies/node_affinity.go

```diff
@@ -70,5 +69,6 @@ func removePodsViolatingNodeAffinityCount(ds *options.DeschedulerServer, strateg
 			return evictedPodCount
 		}
 	}
+	glog.V(1).Infof("Evicted %v pods", evictedPodCount)
```

Comment:
How does it help to move the log message here? It is logged in both cases under the same conditions.

Reply:
previously this msg was being logged even when this strategy was disabled so want to avoid it.

Reply:
I see now:
```
 	if !strategy.Enabled {
 		return evictedPodCount
 	}
```